Scripts/MNISTdataset.swift

```diff
@@ -0,0 +1,97 @@
+/*
+Usage:
+
+downloadAndExtract()
+let (trainX, trainY, testX, testY) = readMNIST(trainImages: constants["trainImages"]!, 
+                                                   trainLabels: constants["trainLabels"]!, 
+                                                   testImages: constants["testImages"]!, 
+                                                   testLabels: constants["testLabels"]!)
+*/
+
+import Python
+import TensorFlow
+
+let np = Python.import("numpy")
+let path = Python.import("os.path")
+let urllib = Python.import("urllib.request")
+let gzip = Python.import("gzip")
+let sh = Python.import("shutil")
+
+// MNIST constants
+let constants: [String: String] = [
+    "URL": "http://yann.lecun.com/exdb/mnist/",
+    "trainImagesFile": "train-images-idx3-ubyte.gz",
+    "trainLabelsFile": "train-labels-idx1-ubyte.gz",
+    "testImagesFile": "t10k-images-idx3-ubyte.gz",
+    "testLabelsFile": "t10k-labels-idx1-ubyte.gz",
+    "trainImages": "train-images-idx3-ubyte",
+    "trainLabels": "train-labels-idx1-ubyte",
+    "testImages": "t10k-images-idx3-ubyte",
+    "testLabels": "t10k-label-idx1-ubyte",
+]
+
+// Function to download and extract files from online
+func downloadAndExtract(){
```

Comment:
If you make `readMNIST` call this automatically, then this can be `fileprivate`.

Put a space between the `)` and `{`.